wordgenerator.py

```diff
@@ -0,0 +1,110 @@
+import pygame
+import random
+import sys
+import math
+from copy import deepcopy
+
+CONST_FILENAME = "american-english.txt"  
+
+'''
+Opens the file CONST_FILENAME and reads entire file to word_array list. Shuffles the array and then sorts according to length
+'''
+class WordGenerator():
+    index_locations = []
```

Comment:
Not sure if you want to change this but I think it'd be good for you to know.

Writing the member variables out of any function without a `self` makes it a class member, not an instance member. Class members are similar to static member variables in c++. 

Initializing instance members should be done in the `__init__` function like:
```
def __init__(self):
    self.index_locations = []
    ...
```